master/internal/webhooks/postgres_webhook.go

```diff
@@ -723,3 +724,25 @@ WHERE q.id = webhook_events_queue.id RETURNING webhook_events_queue.*
 	}
 	return &eventBatch{tx: &tx, events: events}, nil
 }
+
+// updateWebhook updates a webhook in the database.
+func (l *WebhookManager) updateWebhook(
```

Comment:
`updateWebhook` seems not related to `WebhookManager`, maybe we can make it a standalone function?

Reply:
Yeah, it wasn't related to `WebhookManager`. I ended up adding a feature that updates url of webhook in the cache. `if err := l.editTriggers(ts); err != nil {`